gen/templates/resource.go.tmpl

```diff
@@ -15,6 +15,13 @@ import (
 	
 	"github.com/ciscoecosystem/aci-go-client/v2/client"
 	"github.com/ciscoecosystem/aci-go-client/v2/container"
+	{{- range .Properties}}
```

Comment:
Can you try rendering without the loop, I think with a change introduced (https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/main.go#L23) we might not the range and conditional (I am not sure about this and is mainly curiosity, because if this is the case we can actually simplify a lot of the import template part).